pyproject.toml

```diff
@@ -23,14 +23,10 @@ exclude = '''
 )
 '''
 [tool.pyright]
-executionEnvironments=[
-  {"root" = "stripe"},
-  {"root" = "tests"}
-]
```

Comment:
Why don't we need executionEnvironments anymore?

Reply:
@richardm-stripe  will know better, some fix in pyright made it non-required